nixos/xdg-terminal-exec: init module (#303225)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2405.section.md

- [ALVR](https://github.com/alvr-org/alvr), a VR desktop streamer. Available as [programs.alvr](#opt-programs.alvr.enable)

> - [xdg-terminal-exec](https://github.com/Vladimir-csp/xdg-terminal-exec), a proposal for XDG terminal execution utility. Available as [xdg.terminal-exec](#opt-xdg.terminal-exec.enable).

Reviewer: ```suggestion
- [xdg-terminal-exec](https://github.com/Vladimir-csp/xdg-terminal-exec), the proposed Default Terminal Execution Specification. Available as [xdg.terminal-exec](#opt-xdg.terminal-exec.enable).
```
